File: example/spring-boot-graphql/src/main/kotlin/cz/encircled/joiner/springbootgraphql/SpringBootExampleApplication.java
package cz.encircled.joiner.springbootgraphql;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication
public class SpringBootExampleApplication {

    public static void main(String[] args) {
        new SpringApplicationBuilder(SpringBootExampleApplication.class).run(args);
    }

}

File: example/spring-boot-example/src/main/kotlin/cz/encircled/joiner/springbootexample/SpringBootExampleApplication.kt
package cz.encircled.joiner.springbootexample

import ch.vorburger.mariadb4j.springboot.autoconfigure.DataSourceAutoConfiguration
import ch.vorburger.mariadb4j.springboot.autoconfigure.MariaDB4jSpringConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Import


@SpringBootApplication
@Import(DataSourceAutoConfiguration::class, MariaDB4jSpringConfiguration::class)
class SpringBootExampleApplication {

}

fun main(args: Array<String>) {
    runApplication<SpringBootExampleApplication>(*args)
}
